packages/internal-test-env/index.ts

```diff
@@ -435,7 +435,7 @@ export async function getPodRoot(session: Session) {
 
 export function createFetch(session: Session, userAgent: string): typeof fetch {
   return (url, options?) => {
-    return session.fetch(url, {
+    return session.fetch(url as string | Request, {
```

Comment:
Is there any reason not to do `url.toString()` instead of the type cast?

Reply:
Good point - will do that